### PR TITLE
Added express. Closes #1

### LINKED ISSUE #1: Use express
The web server currently used is the standard HTTP node server. Let's switch to Express so that we can have more control over what is served from where without adding a load of boilerplate in.

### PR DESCRIPTION
**Pull request by MadhouseSteve**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,12 +7,21 @@
...
+    "start": "node dist/index.js",
+    "test": "jest"
   },
   "devDependencies": {
+    "@types/express": "^4.17.8",
+    "@types/jest": "^26.0.14",
     "@types/node": "^14.11.2",
+    "jest": "^26.4.2",
     "nodemon": "^2.0.4",
+    "supertest": "^5.0.0",
```
Comment (whitep4nth3r) [inline package.json:19]: If you're going to host on Heroku, put all of these things into dependencies LUL

File: src/index.ts [hunk 2]
```diff
@@ -1,14 +1,10 @@
 import * as http from 'http';
 import { AddressInfo } from 'net';
+import express from './server/express';
 
-const server = http.createServer(handler);
+const server = http.createServer(express);
 server.listen({ port: 3000, host: "127.0.0.1" }, () => {
     const address = server.address() as AddressInfo;
 
     console.log(`🥌 Listening at http://${address.address}:${address.port}`);
```
Comment (whitep4nth3r) [inline src/index.ts:9]: Hahahahahaha is that a CURLING ball?!

File: package.json (hunk 1 above)
Comment (MadhouseSteve) [reply package.json:19]: This is just the framework, and the tests are for making sure the framework makes sense. I'd expect people using the framework to do their own tests 😛

File: src/index.ts (hunk 2 above)
Comment (MadhouseSteve) [reply src/index.ts:9]: I can't remember 😨maybe i should make it a cat 😸

File: package.json (hunk 1 above)
Comment (whitep4nth3r) [reply package.json:19]: Ah yes!

File: src/index.ts (hunk 2 above)
Comment (whitep4nth3r) [reply src/index.ts:9]: My thoughts exactly.

**Pull request merged**